packages/contracts/contracts/ExitModule/ExitBase.sol

```diff
@@ -70,7 +65,7 @@ abstract contract ExitBase is Module, IExitBase, IERC165 {
     // @param amount to transfer
     function transferNativeAsset(address to, uint256 amount) private {
         require(
-            exec(to, amount, bytes("0x"), Enum.Operation.Call),
+            exec(to, amount, bytes(""), Enum.Operation.Call),
```

Comment:
Fix looks good. This will yield the intended empty data buffer.

Super minor optional comment: I'm just not sure I love seeing this written as a string cast. We have three options of writing this:
- bytes("")
- declare "bytes memory data"/ "bytes memory emptyData" and pass in that variable
- new bytes(0)

First and second are cheapest and cost equivalent

Reply:
I have seen `new bytes(0)` and `bytes("")` but can't find any information on which to prefer.

Reply:
`new bytes(0)` appears to be the cheapest, by a small margin, and it feels cleaner than typecasting. Let's roll with that.

Reply:
Really? I ran it on remix, and bytes(0) was the most expensive I think

Reply:
In any case cost diff will be marginal, and this is the cleanest looking form

Reply:
Ah, you're right. Turns out remix changed the order of the functions, so I read it incorrectly earlier.

I wrote this little test contract:
```solidity
contract BytesTest {
    function bytesTypecastString() public pure returns(bytes memory) {
        return bytes("");
    }

    function bytesNewBytes() public pure returns(bytes memory) {
        return new bytes(0);
    }

    function bytesDeclaired() public pure returns(bytes memory) {
        bytes memory data;
        return data;
    }

    function bytesConcat() public pure returns(bytes memory) {
        return bytes.concat();
    }

    function bytesEmptyString()  public pure returns(bytes memory) {
        return "";
    }
}
```

And got the following results:
`bytesTypecastString:` `619` gas
`bytesNewBytes`: `711` gas
`bytesDeclaired`: `633` gas
`bytesConcat`: `676` gas
`bytesEmptyString:` `619` gas

I think my preferred option is the version from `bytesEmptyString` because it looks clean and is equal cheapest with `bytesTypecastString`.

Reply:
Ah okay I hadn't considered the implicit cast option. Yes agree, empty string (without cast) is the cleanest. I would also pick that one